### PR TITLE
QA and CI sections

### PR DESCRIPTION
**Pull request by swilliams**
Adds some more details on what QA should do, and what CI does (this still needs more color).

### CODE REVIEW

File: docs/Quality.md [hunk 1]
```diff
@@ -43,11 +43,36 @@ Utilities like [Istanbul](https://github.com/gotwarlost/istanbul) provide the pe
 	* [BEHAT](https://www.drupal.org/project/drupalextension)
 	* [phpunit](https://phpunit.de)
 
-## QA?
-TODO
+## QA
+QA can take different forms. At the most basic level, a QA team can perform manual testing. This involves writing manual scripts, executing them, and then filing bugs when issues are encountered while executing those scripts. Manual testing is time consuming, and is still prone to human error.
+
+A QA team can also write automated tests, even down to the unit level. However, developers tend to be best suited for unit and most integration testing, since those benefit from a deeper understanding of what will be written. A QA Engineer is well suited for writing and maintaining end-to-end tests though, since those focus on whether acceptance criteria has been satisified or not.
```
Comment (TinaHeiligers) [inline docs/Quality.md]: "...acceptance criteria has been..." should read: "...acceptance criteria have been..." assuming criteria is the plural form of 'criterion'

File: docs/Quality.md [hunk 2]
```diff
@@ -43,11 +43,36 @@ Utilities like [Istanbul](https://github.com/gotwarlost/istanbul) provide the pe
...
+* Steps to reproduce the issue.
+* How reproducible the issue is.
+* Severity
+  * Showstopper (this breaks major pieces of functionality)
+  * Major (a big problem, but not a critical fix)
+  * Minor (affects a smaller piece functionality)
+  * Trivial (eg. a copy error)
+
+### Bug Triage
+Bugs should be treated similarly as issues and scheduled as a part of Sprint grooming and planning. Showstoppers and certain Major issues will likely require immediate attention. All other severities should be mixed in with stories and scheduled appropriately.
```
Comment (TinaHeiligers) [inline docs/Quality.md]: "...similarly as issues..." should read "...similarly to issues..."

Comment (TinaHeiligers) [reply docs/Quality.md]: I've been told that the use of 'grooming the backlog' is not encouraged and should be replaced with 'backlog refinement'. Personally, I don't have a preference.

**Comment by krantzinator** [comment]
👍

**Comment by mmislam101** [comment]
Merging this so I can add to it. Will implement suggestions in my PR

**Pull request merged**